quarkus-picocli/src/test/java/PicocliProfileTestIT.java

```diff
@@ -15,71 +16,54 @@
 import io.quarkus.ts.qe.services.AgeService;
 import io.quarkus.ts.qe.services.HelloService;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @QuarkusScenario
 public class PicocliProfileTestIT {
 
     @QuarkusApplication(classes = { AgeCommand.class, CommonOptions.class, EntryCommand.class, HelloCommand.class,
             OtherCommand.class, OtherEntryCommand.class,
             Configuration.class, AgeService.class, HelloService.class }, properties = "test.properties")
-    static final RestService app = new RestService().setAutoStart(false);
+    static final RestService app = new RestService().withProperty("quarkus.args", "age --age 30");
```

Comment:
I'll put it `.withProperty("quarkus.args", "age --age 30");` inside of the test method if you agree.

Reply:
but I recommend to make no changes to this PR until we have FW rebased on this. this PR I know works, so refactoring can be done on this PR when everything is green. I'll leave refactoring to you.